tools/build.js

```diff
@@ -868,6 +878,17 @@ task(function all() {
   }
 });
 
+// Install POM tasks
+if ( TASKS ) {
+  TASKS.forEach(f => {
+    if ( tasks.indexOf(f.name) > -1 ) {
+      task(f);
+    } else {
+      globalThis[f.name] = f;
```

Comment:
Why not always make a task()?

Reply:
I would think that except for before and after tasks you would want to list them with the other -X options.

Reply:
Just don't add before_ and after_ tasks to the tasks array.